app/controllers/all-service-transactions/get.controller.test.js

```diff
@@ -53,6 +54,37 @@ describe('All service transactions - GET', () => {
     })
   })
 
+  describe('Error results when from date is later than to date', () => {
+    const request = {
+      account: gatewayAccountFixture.validGatewayAccount({ 'payment_provider': 'stripe' }),
+      service: service,
+      user: user,
+      params: {},
+      query: {
+        fromDate: '03/5/2018',
+        fromTime: '01:00:00',
+        toDate: '01/5/2018',
+        toTime: '01:00:00'
+      },
+      url: 'http://selfservice/all-servce-transactions',
+      session: {}
+    }
+    const response = {
+      render: sinon.spy()
+    }
+
+    it('should return the response with the date-range failing validation with empty transaction results indicator', async () => {
+      await getController()(request, response, next)
+
+      sinon.assert.called(response.render)
+      expect(response.render.firstCall.args[0]).to.equal('transactions/index')
```

Comment:
This is something that you would not have known - but we prefer to use `sinon.assert` now.
We have a manual page about this:
https://manual.payments.service.gov.uk/manual/coding-standards/node-js-testing-strategy.html#assertions